<?xml version="1.0"?>
<launch>

    <arg name="rviz" default="true" />
    <arg name="project" default="multi_sensor_mapping"/>

    <!-- Lidar IMU Mapper -->
    <node pkg="$(arg project)" type="cloud_viewer" name="cloud_viewer" output="screen" respawn="false">
        <param name="cloud_path" value="/home/hkw/global_cloud.pcd"/>
    </node>


    <!--- Rviz -->
    <!-- <group if="$(arg rviz)">
      <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find multi_sensor_mapping)/rviz/lidar_mapper.rviz" />
    </group> -->
</launch>
